test/InputFilter/RestService/PatchInputFilterTest.php

```diff
@@ -198,10 +134,46 @@ public function dataProviderIsValidFalse(): array
 
     /**
      * @dataProvider dataProviderIsValidTrue
-     * @param array<string, mixed> $data
+     * @param mixed $pageSize
```

Comment:
```suggestion
     * @param string|int $pageSize
```

This should match the provider.